src/EventListener/RequestConfigurationSubscriber.php

```diff
@@ -91,4 +91,15 @@ public function onKernelController(ControllerEvent $event): void
         $request->setRequestFormat('json');
         $request->attributes->set(self::ATTRIBUTE_API, true);
     }
+
+	private function getAnnotation(\ReflectionMethod $reflection, string $annotationClass)
+	{
+		if (\PHP_VERSION_ID >= 80000) {
+			foreach ($reflection->getAttributes($annotationClass) as $attribute) {
+				return $attribute->newInstance();
```

Comment:
In case of moving this method into a separate class, then it will be good to cover situation when bunch of attributes or annotations of the same class can be returned from it

Reply:
bunch or single can be returned tested in php
https://www.php.net/manual/en/language.attributes.classes.php#example-312

but  I agreed that this point should be used more transparently in the code